app/src/main/java/com/android/ubclaunchpad/driver/UI/sessionInfoActivity.java

```diff
@@ -7,109 +7,170 @@
 import android.widget.ListView;
 import android.widget.TextView;
 import com.android.ubclaunchpad.driver.R;
+import com.android.ubclaunchpad.driver.models.SessionModel;
+import com.android.ubclaunchpad.driver.models.User;
 import com.android.ubclaunchpad.driver.util.FirebaseUtils;
+import com.android.ubclaunchpad.driver.util.StringUtils;
+import com.android.ubclaunchpad.driver.util.UserManager;
 import com.google.firebase.database.ChildEventListener;
 import com.google.firebase.database.DataSnapshot;
 import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.ValueEventListener;
+
 import java.util.ArrayList;
 
 public class sessionInfoActivity extends AppCompatActivity {
 
     private static final String TAG = "sessionInfoActivity";
+    final String passengerDistance = "\nP\n\t\t\t\t";
+    final String driverDistance = "\nD\n\t\t\t\t";
+    final ArrayList<String> itemsArray = new ArrayList<String>();
    // private DatabaseReference mDatabase;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_session_info);
 
-        final ArrayList<String> itemsArray = new ArrayList<String>();
         final ListView listView = (ListView) findViewById(R.id.sessionItemsList);
         final ArrayAdapter<String> adapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, android.R.id.text1, itemsArray);
         listView.setAdapter(adapter);
-        String sessionName = getIntent().getStringExtra(getString(R.string.session_name));
-        final String passengerDistance = "\nP\n\t\t\t\t";
-        final String driverDistance = "\nD\n\t\t\t\t";
+        final String sessionName = getIntent().getStringExtra(getString(R.string.session_name));
```

Comment:
You didn't do this but can we move the session_name constant into a String constant in StringUtils? Like public static final String SESSION_NAME = -whatever the constant value is-. Strings in R.string are supposed to be localized (varies from region to region depending on the region's language) and can actually change depending on your location.